ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activitySection.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="margin-bottom" *ngFor="let block of section.blocks">
+    <ng-container *ngIf="block.displayNumber; then numbered else notNumbered">
+    </ng-container>
+
+    <ng-template #numbered>
+        <ol class="ddp-list">
+            <li class="ddp-li" [value]="block.displayNumber">
+                <ng-container *ngTemplateOutlet="content"></ng-container>
+            </li>
+        </ol>
+    </ng-template>
+
+    <ng-template #notNumbered>
+        <div class="ddp-single-question">
+            <ng-container *ngTemplateOutlet="content"></ng-container>
+        </div>
+    </ng-template>
+
+    <ng-template #content>
+        <div *ngIf="isConditional(block)">
+            <ddp-conditional-block [block]="block"
+                                   [readonly]="readonly"
+                                   [validationRequested]="validationRequested"
+                                   [studyGuid]="studyGuid"
+                                   [activityGuid]="activityGuid">
+            </ddp-conditional-block>
+        </div>
+        <div *ngIf="isQuestion(block)">
+            <ddp-activity-question [block]="block"
+                                   [readonly]="readonly"
+                                   [validationRequested]="validationRequested"
+                                   [studyGuid]="studyGuid"
+                                   [activityGuid]="activityGuid"
+                                   (componentBusy)="componentBusy.emit($event)">
+            </ddp-activity-question>
+        </div>
+        <div *ngIf="isInstitution(block)">
+            <ddp-institutions-form [block]="block"
+                                   [studyGuid]="studyGuid"
+                                   [readonly]="readonly"
+                                   [validationRequested]="validationRequested"
+                                   (validStatusChanged)="updateValidationStatusInSection(block.id, $event)"
```

Comment:
Renamed `updateValidationStatus` => `updateValidationStatusInSection`
Passing id as a key (instead of hard-coded numbers) 

The same for lines 51, 75.